<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smFontFeaturesBase</class>
 <widget class="QWidget" name="smFontFeaturesBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>488</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#008000;&quot;&gt;The current selected font does not have any font features.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ligaturesGroupBox">
     <property name="title">
      <string>Ligatures</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="CommonCheck">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Common Ligatures (liga)</string>
        </property>
        <property name="text">
         <string>Common</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="DiscretionaryCheck">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Discretionary Ligatures (dlig)</string>
        </property>
        <property name="text">
         <string>Discretionary</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="ContextualCheck">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Contextual Ligatures (clig)</string>
        </property>
        <property name="text">
         <string>Contextual</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="HistoricalCheck">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Historical Ligatures (hlig)</string>
        </property>
        <property name="text">
         <string>Historical</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="capitalsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Capitals &amp;&amp; Position</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QComboBox" name="capitalsComboBox">
          <item>
           <property name="text">
            <string>Default Capitals</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Small Capitals</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Petite Capitals</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Unicase</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Titling</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Petite Capitals from Capitals</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Small Capitals from Capitals</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="capitalsLabel">
          <property name="text">
           <string>Capitals:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="capitalsSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="positionLabel">
          <property name="text">
           <string>Position:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="positionComboBox">
          <item>
           <property name="text">
            <string>Default Position</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Subscript</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Superscript</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ordinals</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="positionSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="numeralsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Numerals</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QComboBox" name="numeralComboBox">
          <item>
           <property name="text">
            <string>Default Numerals</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Lining</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Old Style</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="numeralSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="styleLabel">
          <property name="text">
           <string>Style:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="widthLabel">
          <property name="text">
           <string>Width:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="widthSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="widthComboBox">
          <item>
           <property name="text">
            <string>Default Numeral Width</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Proportional</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tabular</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="fractionLabel">
          <property name="text">
           <string>Fraction:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="fractionComboBox">
          <item>
           <property name="text">
            <string>No Fractions</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Diagonal Fractions</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Stacked Fractions</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="2">
         <spacer name="fractionSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="SlashedZeroCheck">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Slashed Zero (zero)</string>
        </property>
        <property name="text">
         <string>Slashed Zero</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="stylisticSetsLabel">
       <property name="text">
        <string>Stylistic Sets:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stylisticSetsButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>168</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>click to select</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
